.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -169,7 +169,7 @@ jobs:
 
       - name: Run e2e test suite
         shell: bash
-        timeout-minutes: 10
+        timeout-minutes: 20
```

Comment:
Sorry for later commit, but not sure if would help anything - from my (brief) experience if we stucked even for one minute, waiting 19 more would not change anything - there is some problem with e2e tests that if we run them for the second time on the **same** set of aleph-node instances in docker (ie without having fresh instances every time) then we sometimes end up stuck. 

I would say we'd need to revise as separate task what happens in e2e suite so we can run them repeatedly on sme instances of aleph-node containers.